Helper/FeatureSwitch/Decider.php

```diff
@@ -156,7 +161,19 @@ public function isSwitchEnabled($switchName) {
      * Switch Helpers below
      ***************************************************/
 
-    public function isSampleSwitchEnabled() {
+    /**
+     * Checks whether the sample feature switch is enabled
+     */
```

Comment:
Please add `@return bool` to PHPDoc Comment

Reply:
@akulkapoor this please :)